ProjFS.Linux/PrjFSLib.Linux.Managed/VirtualizationInstance.cs

```diff
@@ -36,15 +38,23 @@ public static Result ConvertDirectoryToVirtualizationRoot(string fullPath)
                 OnNotifyOperation = this.preventGCOnNotifyOperationDelegate = new NotifyOperationCallback(this.OnNotifyOperation),
             };
 
-            return Interop.PrjFSLib.StartVirtualizationInstance(
+            IntPtr newMountHandle = IntPtr.Zero;
```

Comment:
I don't think you need a new variable here. Just use `mountHandle`?

Reply:
Yup, that works -- thanks, it's cleaner now.  Still learning the C# stuff!  :-)